model/src/main/kotlin/utils/OrtResultExtensions.kt

```diff
@@ -84,3 +90,35 @@ fun OrtResult.getDetectedLicensesWithCopyrights(
     omitExcluded: Boolean = true
 ): Map<String, Set<String>> =
     LicenseResolver(this, packageConfigurationProvider).getDetectedLicensesWithCopyrights(id, omitExcluded)
+
+/**
+ * Create [PackageConfiguration]s for the projects in this [OrtResult]. [PathExclude]s and [LicenseFindingCuration]s are
+ * taken from the [RepositoryConfiguration]. The [Provenance] is taken from the scan results. Projects without scan
+ * result and scan results without provenance information are ignored.
+ */
+fun OrtResult.getProjectConfigurations(): List<PackageConfiguration> {
```

Comment:
Why did you decide to ignore projects without scan result? Wouldn't it be simpler to return exactly one PackageConfiguration for each Project contained in the OrtResult?

Reply:
Because the package configuration needs to match the provenance of the scan result, and without scan result the package configuration is useless.

Reply:
A project actually has both, `sourceArtifactUrl` and `resolveVcsInfo` so it is possible to construct a matching provenance.
Whether it is useless or not is determined by the caller of this function, but not by the function itself.  
I'd still prefer to do a 1:1 mapping from Project <-> Configuration for simplicity. I'll leave decision up to you.

If the current approach is kept, it needs to be adjusted to ensure that the `matcher` is unique for the returned list of package configurations. Consider e.g. different scanner details, duplicated scan results and so on.

Reply:
Done.